resolwe_bio/processes/reads_processing/xengsort.py

```diff
@@ -38,15 +38,6 @@ def concatenate_files(filenames, out_fname, decompress=False):
                 )
 
 
-def create_filename(basename, suffix):
-    """Create a secure version of a filename."""
-    # Keep only alphanumeric characters and underscores.
-    basename = re.sub(pattern=r"[\W]", repl="_", string=basename)
-    # Replace consecutive underscores with a single one.
-    basename = re.sub(pattern=r"(_)\1+", repl=r"\1", string=basename)
-    return f"{basename}.{suffix}"
-
-
 def concatenate_reads(filenames, out_fasta, error):
     """Concatenate reads in FASTQ files."""
     try:
```

Comment:
Concatenation could be made faster by disabling decompression which is no longer needed for the new version of Xengsort. `decompress=True` can be changed to False one line below. Mate file names should then be modified accordingly.